kubernetes/gke-prow/prow/kustomization.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: default
+
+resources:
```

Comment:
Just a nit: would be nice to sort this file A-Z, `gateway.yaml` and `monitoring.yaml` are in the wrong place.